src/client/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import React from 'react'
+import { format } from 'date-fns'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useTextKeys } from 'utils/textKeys'
+import { Telephone } from '../icons/Telephone'
+
+type Props = {
+  color?: 'black' | 'white'
+}
+
+const Wrapper = styled.div<Props>`
+  color: ${(props) =>
+    props.color === 'black' ? colorsV3.black : colorsV3.white};
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  && {
```

Comment:
`&` is a reference to the class name of the styled component itself. By adding additional `&&` you can increase the CSS specificity of the rule to override e.g. global styles.

Reply:
🤔 I see. But why do we need this here? Are we concerned with some global style rule that sets `margin` for regular  `div`s?

Reply:
I think in general it's good to avoid `&&` if it's not for some strange reason strictly necessary. It mostly just raises a lot of questions and makes it harder to understand what style will actually be applied to what. And I truly hope that we don't have any global rules setting `margin` for `div`s since that would be crazy 🤪

Reply:
> And I truly hope that we don't have any global rules setting margin for divs since that would be crazy

That's my point 😆 
I've got my questions answered so gonna resolve this. Thanks guys 🙇🏻